scm-webapp/src/test/java/sonia/scm/api/v2/resources/SourceRootResourceTest.java

```diff
@@ -24,7 +24,6 @@
 
 package sonia.scm.api.v2.resources;
 
-import com.google.inject.util.Providers;
```

Comment:
I like the overall cleanup of this class, but somewhere you could add a further test method that checks something like

```java
verify(browseCommandBuilder).setCollapse(true)
```